templates/project/.gitattributes.twig

```diff
@@ -4,5 +4,6 @@
 
 .* export-ignore
 *.md export-ignore
+static-analysis/ export-ignore
```

Comment:
I would prefer a more generic term like `dev`, or `ci` as suggested by @greg0ire.
It will be more re-usable.